network/src/commonTest/kotlin/com/wire/kalium/api/tools/json/api/user/login/SSOLoginApiTest.kt

```diff
@@ -15,14 +15,14 @@ import kotlin.test.assertEquals
 import kotlin.test.assertIs
 
 @OptIn(ExperimentalCoroutinesApi::class)
-class SSOLoginApiTest: ApiTest {
+class SSOLoginApiTest : ApiTest {
 
     @Test
     fun givenBEResponseSuccess_whenCallingInitiateSSOEndpointWithNoRedirect_thenRequestConfiguredCorrectly() = runTest {
         val uuid = "uuid"
         val param = SSOLoginApi.InitiateParam.WithoutRedirect(uuid)
         val expectedPath = "$PATH_SSO_INITIATE/$uuid"
-        val httpClient = mockAuthenticatedHttpClient(
+        val networkClient = mockUnauthenticatedNetworkClient(
```

Comment:
Here, for example, we were passing the wrong `HttpClient`.

This is one of the reasons for this PR.